Makefile

```diff
@@ -124,6 +124,9 @@ e2e/operator-registry: ## Run e2e registry tests
 e2e/olm: ## Run e2e olm tests
 	$(MAKE) e2e WHAT=operator-lifecycle-manager
 
+e2e/ocp:
+	go test -v -mod=vendor -timeout 70m ./test/...
```

Comment:
I had that same thought locally and ended up separating out the Makefile targets:

- This target more-or-less resembles something like `e2e/metrics` but it could instead be generalized upfront so we can easily iterate (without having to also update o/release)
- The OLM testing suite is long and prone to errors/flakes/etc.